src/gridtools/clang/storage_runtime.hpp

```diff
@@ -94,5 +94,14 @@ namespace gridtools {
         /** @} */
 
         /** @} */
+
```

Comment:
This feels like a bad way of doing it. Do you have a better idea?

Reply:
We choose to leave it as is since once the newest gridtools boudary-condition syntax rolls out we need to rework this anyways (and this will presumably be soon)